plutus-pab/src/Plutus/PAB/Core/ContractInstance.hs

```diff
@@ -198,7 +198,11 @@ processTxStatusChangeRequestsSTM =
     where
         handler txId = do
             env <- ask
-            pure (AwaitTxStatusChangeResp txId <$> InstanceState.waitForTxStatusChange Unknown txId env)
+                     -- This should never be called in a context where there is a
+                     -- Marconi indexer in the environment.
```

Comment:
Fix indentation for the comment.

Also, I don't understand the "why" from the comment.